Product (#30)
hostmobility/documentation: docs/product/mx4-c61.md

---
title: Platform MX-4 C61
tags:
  - C61
---
## Abstract

> MX-4 C61 is a small size and low cost telematics computer for automotive development.

Reviewer: ```suggestion
MX-4 C61 is a small form factor and low cost telematics computer for remote diagnostics and fleet management.
```

## Feature summary for HM005-1

> See [Production Variant Definition](#production-variant-definition)

Reviewer: ```suggestion
See [Production variant definition](#production-variant-definition).
```

| Feature | Summary
|------------|-----------------------------------------------------|

> | CPU (main) | ARM Cortex-A5 CPU, 500 MHz single core              |
> | CPU        | ARM Cortex-M4 CPU, 167 MHz single core              |

Reviewer: ```suggestion
| CPU (main) | ARM Cortex-A5, 500 MHz single core              |
| CPU        | ARM Cortex-M4, 167 MHz single core              |
```

> | CoProcessor        | PIC24              |

Reviewer: ```suggestion
| Coprocessor        | PIC24              |
```

| 8   | GND                                                | Reference for INPUT-POWER                                 |
| 9   | DIG-IO-1                                           | Output with source and sink and/or analog in and/or digital in |
| 10  | RS-485 or J1708                                    | RS-485 TX or J1708A (default variant is RS-485 not J1708) |
| 11  | DIG-IO-2                                           | Output with source and sink and/or analog in and/or digital in |
| 12  | RS-485 or J1708                                    | RS-485 RX or J1708B (default variant is RS-485 not J1708) |
| 13  | START-SIGNAL                                       | Must be tied to INPUT-POWER for the unit to start          |
| 14  | RS-232_RXD                                         | Rs-232 receive from pc (default debug)                    |
| 15  | RS-232_TXD                                         | Rs-232 transmit from device to pc (default debug)         |
| S   | GND                                                | The shield acts as a general-purpose GND                   |

### Other connector and buttons

- Fakra red LTE/4G
- Fakra blue GPS

> - Rj45

Reviewer: ```suggestion
- RJ45
```

- Audio & mic
- Reset button
- [LEDs](../interfaces/mx4/leds.md)

## Technical specification

> Contact [support](../support.md) to receive technical specification documentation (1301-HM005-Technical-description) for this hardware.

Reviewer: ```suggestion
Contact [support](../support.md) to receive the technical specification (1301-HM005-Technical-description) for this hardware.
```
